src/vivarium_gates_nutrition_optimization_child/model_specifications/branches/scenarios.yaml

```diff
@@ -1,9 +1,9 @@
-input_draw_count: 25
-random_seed_count: 60
+input_draw_count: 10
+random_seed_count: 10
+
+input_draws: [602, 357, 946, 829, 650, 232, 394, 680, 629, 29]
```

Comment:
Why these specific draws/seeds?

Reply:
These are the draws and seeds in the input child data from the pregnancy model,